README.md

```diff
@@ -1 +1,77 @@
-# jest-tcm-reporter
\ No newline at end of file
+[![Build Status](https://travis-ci.org/optimaxdev/jest-tcm-reporter.svg?branch=master)](https://travis-ci.org/optimaxdev/jest-tcm-reporter)
+# jest-tcm-reporter
+
+This package sends reports to the JTM
+
+## Installation
+
+```bash
+$ yarn add --dev optimaxdev/jest-tcm-reporter#master
+```
+
+## Configuration
+
+Create config file `jest-tcm.config.js`
+
+```js
+// jest-tcm.config.js
+
+module.exports = {
+    tcm: {
+        user: {
+            username: 'user',
+            password: 'password',
+        },
+        projectKey: 'KEY'
+    },
+    reportsFile: {
+        JSON: './public/reports/team-city.json'
+    },
+    url: {
+        host: 'https://jira.example.com/rest/kanoahtests/1.0'
+    }
+}
+```
+
+or add a config to `package.json`
+
+```json
+// package.json
+
+{
+    "jest-tcm": {
+        "tcm": {
+            "user": {
+                "username": "user",
+                "password": "password",
+            },
+            "projectKey": "KEY"
+        },
+        "reportsFile": {
+            "JSON": "./public/reports/team-city.json"
+        },
+        "url": {
+            "host": "https://jira.example.com/rest/kanoahtests/1.0"
+        }
+    }
+}
+```
+
+#### Options
+
+| Variable  | Default Value | Description                                     |
+| :-------- | :------------ | :---------------------------------------------- |
+| cycleName |               | Name cycle in JTM                      |
+| target    |               | Each `cases` separately or group by `describes` |
+
+## Usage
+
+Each case separately
+```bash
+$ npx jest-tcm-reporter --cycleName=ANT_QA-34_Mobile --target=case
+```
+####
+Grouped by describe
+```bash
+$ npx jest-tcm-reporter --cycleName=ANT_QA-34_Mobile --target=describe
+```
```

Comment:
I think you need to add:
- for which Jira extensions is this library used;
- describe that it works only with jest report in JSON format.